src/test/regress/specs/isolation_add_node_vs_copy_reference_table.spec

```diff
@@ -0,0 +1,88 @@
+# initially remove all existing nodes and then
+# add single one of the nodes for the purpose of the test
+setup
+{	
+	SELECT master_remove_node(nodename, nodeport) FROM pg_dist_node;
```

Comment:
I think it's ok to assume tests do a good job and leave an empty `pg_dist_node`. Removing this will help make sure.